web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -509,6 +510,162 @@ const EncryptionPasswordField = ({
   );
 };
 
+/**
+ * Form for configuring the space policy.
+ * @component
+ *
+ * @param {object} props
+ * @param {string} props.id - Form ID.
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ * @param {onSubmitFn} [props.onSubmit=noop] - On submit callback.
+ *
+ * @callback onSubmitFn
+ * @param {string} policy - Name of the selected policy.
+ */
+const SpacePolicySettingsForm = ({
+  id,
+  settings,
+  onSubmit: onSubmitProp = noop
+}) => {
+  const [spacePolicy, setSpacePolicy] = useState(settings);
+
+  const selectDelete = () => {
+    setSpacePolicy("delete");
+  };
+
+  const selectResize = () => {
+    setSpacePolicy("resize");
+  };
+
+  const selectKeep = () => {
+    setSpacePolicy("keep");
+  };
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+    onSubmitProp(spacePolicy);
+  };
+
+  return (
+    <Form id={id} onSubmit={onSubmit}>
+      <div className="split">
+        <span>{_("Policy to make enough free space")}</span>
+        <ToggleGroup isCompact>
+          <ToggleGroupItem
+            text={_("Delete Anything")}
```

Comment:
```suggestion
            text={_("Delete Everything")}
```

Help text?

> All contents (*) will be deleted and lost.
> Make sure that the correct target disk is selected, that you have copied any data that you might still need from the target disk(s).

For (*), if this applies to some disks only, insert
> of the target disk `/dev/sda`